cobra/test/test_core/test_core_reaction.py

```diff
@@ -269,6 +269,13 @@ def test_build_from_string(model):
         assert model.metabolites.foo
         assert len(model.metabolites) == m
 
+    with model:
+        config = Configuration()
```

Comment:
Since you anyway have to fix the test. I'd write it slightly differently.

1. Make `config` global because it is a global singleton. (Bad for testing but this is the design choice.)
2. Then test with different bounds.
```py
old_bounds = config.bounds
config.bounds = -5, 5
pgi.build_reaction_from_string("g6p_c = f6p_c + new", verbose=False)
assert pgi.bounds == (-5, 5)
config.bounds = old_bounds
```